README.md

```diff
@@ -87,3 +87,59 @@ To delete the container run:
 ```
 docker container rm [container_id]
 ```
+
+## Troubleshooting
+
+As you run the website locally, you may run into the following error:
+
+```
+➜ hugo server
+
+INFO 2021/01/21 21:07:55 Using config file: 
+Building sites … INFO 2021/01/21 21:07:55 syncing static files to /
+Built in 288 ms
+Error: Error building site: TOCSS: failed to transform "scss/main.scss" (text/x-scss): resource "scss/scss/main.scss_9fadf33d895a46083cdd64396b57ef68" not found in file cache
+```
+
+To solve this error, first install `hugo-extended` as:
+
+```
+npm install hugo-extended --save-dev
+# or...
+yarn add hugo-extended --dev
+```
+
```

Comment:
I think from this line down we've moved out of the realm of troubleshooting (or even anything Docsy-specific) and just telling users how to run hugo using npm and their `package.json` file, which may not be the preferred approach of all our users.

Reply:
This is still very useful content though so I'm going to add it to our user guide rather than having it here.